content_scripts/courseReverseOrder.js

```diff
@@ -0,0 +1,52 @@
+let DEGREES = 0;
+const DEGREES_TO_REVERSE = 180;
+
+//TODO: Move CSS to CDN.
+const addButtonStyle = () => {
+    const style = document.createElement('style');
+    style.textContent = `
+    .reverse-button{
+        background-color: #269CE9;
+        border: none;
+        color: white;
+        padding: 6px 28px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 14px;
+        border-radius: 5px;
+        font-weight: bold;
+      }
+      
+    .reverse-button:hover{
+        background-color: #70B9E8;
+    }
+    
+    .reverse-button:active{
+        background: #269CE9;
+    }
+    `;
+    document.head.append(style);
+}
+
+const addCourseReverse = () => {
+    let courseContent = document.getElementsByClassName("course-content");
+    if (courseContent) {
+        addButtonStyle();
+        let topDiv = courseContent[0];
+        var reverseCourseBtn = document.createElement("button");
+        reverseCourseBtn.innerHTML = 'Reverse topics order';
+        reverseCourseBtn.className = 'reverse-button';
+        reverseCourseBtn.addEventListener('click', async function () {
+            DEGREES += DEGREES_TO_REVERSE;
+            document.getElementsByClassName("topics")[0].style.transform = `rotate(${DEGREES}deg)`;
```

Comment:
I think this is safer because I have a course (Namely "software testing") which the element is called `"weeks"` instead of `"topics"`, so this fix should do it. WDYT?
```suggestion
            document.getElementsByClassName("section main clearfix")[0].parentElement.style.transform = `rotate(${DEGREES}deg)`;
```

Reply:
Weird, worked with all my courses... But if you say not all lists have class named `topics` then maybe we should go with your suggestion. Good catch!